extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj

```diff
@@ -8,7 +8,7 @@
     <!--Version information-->
     <VersionPrefix>2.0.0</VersionPrefix>
     <VersionSuffix>-preview4</VersionSuffix>
```

Comment:
Do we need to remove preview today as well? Or is this for a different PR?

Reply:
it's a couple PRs away (we have to release the Worker/Worker.Core stuff first, then take that as a dependency here)